topostats/run_topostats.py

```diff
@@ -228,16 +228,30 @@ def main(args=None):
 
     with Pool(processes=config["cores"]) as pool:
         results = defaultdict()
+        image_stats_df_list = []
```

Comment:
The `results` is made a dictionary so that its possible to keep track of what image the `result` comes from when it comes to `pd.concat()` the results.

I think this would perhaps be useful to keep track of too so...

```suggestion
        image_stats = defaultdict()
```